pubsub_nats_options.go

```diff
@@ -126,6 +131,9 @@ func NATSOptRespondToChannel(ctx context.Context, resp chan *Publication) PubSub
 	}
 }
 
+// DEPRECATED: use NATSOptPublishRequireAck and/or NATSOptRespondToChannel instead;
+// this option will be removed in a future release
+//
```

Comment:
🔥 this is useless now

Reply:
I could get rid of it now as we are internally not using it anywhere, but wanted to double check. It is technically a breaking change if we delete it so 🤷‍♂

Reply:
it's not used anywhere AFAIK, delete it